tests/validation/trace_analysis.hpp

```diff
@@ -21,6 +21,11 @@ namespace mc {
 
 double linf_distance(const trace_data& u, const trace_data& ref);
 
+// Compute linf distance as above, excluding samples near
```

Comment:
The comments above function definitions is inconsistent. in `trace_analysis.cpp`
```
/*
 * Description of function
 */
```
Then here we are using single `//` comments.
Elsewhere we have used `///` to indicate a descriptive comment above functions (and their prototypes).
I certainly prefer that to the 'noisy' style in `trace_analysis.cpp`.